tabbycat/participants/models.py

```diff
@@ -101,6 +101,9 @@ class Person(models.Model):
     anonymous = models.BooleanField(default=False,
         verbose_name=_("anonymous"),
         help_text=_("Anonymous persons will have their name and team redacted on public tab releases"))
+    code_name = models.CharField(max_length=25, blank=True, null=True,
+        verbose_name=_("code name"),
+        help_text=_("Name used to obscure real name on public-facing pages"))
```

Comment:
What's the intended behaviour if `self.code_name` isn't set? Currently it returns `None`, which can cause TypeErrors in places that are expecting a string. Examples (last two lines of stack trace only):

- when loading path bp88team/admin/participants/adjudicator/1/
  ```
  File "/home/czlee/tabbycat/tabbycat/utils/tables.py", line 591, in add_debate_adjudicators_column
    'text': construct_text(adjs_data),
  File "/home/czlee/tabbycat/tabbycat/utils/tables.py", line 537, in construct_text
    adj_str = '<span class="d-inline">' + a['adj'].get_public_name(self.tournament)
  ```
- when loading path bp88team/participants/adjudicator/1/:
  ```
  File "/home/czlee/tabbycat/tabbycat/utils/tables.py", line 913, in add_debate_results_columns
    cell = self._team_cell(team, show_emoji=False, subtext=subtext)
  File "/home/czlee/tabbycat/tabbycat/utils/tables.py", line 287, in _team_cell
    cell['popover']['content'].append({'text': ", ".join(speakers)})
  ```

I don't think we need to support cases where code names aren't properly set of course (or I guess more precisely, we definitely shouldn't fall back to real name!), but would it be easier to leave out `null=True` and just use `blank=True`, so that if a code name isn't set, it reverts to the empty string?

Reply:
True